### PR TITLE
Add LoadBalancer details to OSD docs

### PR DESCRIPTION
**Pull request by wgordon17**

### CODE REVIEW

**Comment by wgordon17** [comment]
Staging deployed at: http://httpd-wgordon-docs.e8ca.engint.openshiftapps.com/add-lb-osd-docs/getting_started/access_your_services.html

**Comment by wgordon17** [comment]
@blrm Can I get a technical review of http://httpd-wgordon-docs.e8ca.engint.openshiftapps.com/add-lb-osd-docs/getting_started/access_your_services.html#expose-tcp-services?

**Comment by wgordon17** [comment]
@spurtell Can I get a content review?

**Comment by blrm** [comment]
@wgordon17 :+1: lgtm

**Review by blrm — APPROVED** [review]

**Comment by wgordon17** [comment]
Hey @ahardin-rh this is pending a new subscription offering, so can't merge this yet. But once the SKU goes live, this will be ready to go 👍

File: getting_started/access_your_services.adoc [hunk 1]
```diff
@@ -113,6 +113,74 @@ Alternatively, you can use a custom domain:
 $ oc expose service <service-name> -l route=external --name=<custom-route-name> --hostname=<custom-domain>
 ----
 
+[[expose-tcp-services]]
+== Expose TCP Services
+
+{product-title} xref:../architecture/networking/routes.adoc[routes] expose applications by proxying traffic through HTTP/HTTPS(SNI)/TLS(SNI)
```
Comment (ahardin-rh) [inline getting_started/access_your_services.adoc]: We need to include the section ID at the top of the page.

xref:../architecture/networking/routes.adoc#architecture-core-concepts-routes[routes]

xref:../architecture/core_concepts/pods_and_services.adoc#architecture-core-concepts-pods-and-services[pods and services]

File: getting_started/access_your_services.adoc [hunk 2]
```diff
@@ -113,6 +113,74 @@ Alternatively, you can use a custom domain:
...
+to xref:../architecture/core_concepts/pods_and_services.adoc[pods and services]. A link:https://kubernetes.io/docs/concepts/services-networking/#loadbalancer[LoadBalancer]
+service creates an AWS Elastic Load Balancer (ELB) for your {product-title} cluster, enabling direct TCP access to applications exposed by your LoadBalancer service.
+
+[NOTE]
+====
+LoadBalancer services require an additional purchase. Reach out to your sales team if you're interested in using LoadBalancer services for your {product-title} cluster.
```
Comment (ahardin-rh) [inline getting_started/access_your_services.adoc]: s/Reach out to your sales team if you're/Contact your sales team if you are

File: getting_started/access_your_services.adoc [hunk 3]
```diff
@@ -113,6 +113,74 @@ Alternatively, you can use a custom domain:
...
+====
+
+=== Check your LoadBalancer Quota
+
+By purchasing LoadBalancer services, you're provided with a quota of LoadBalancers available for your {product-title} cluster.
```
Comment (ahardin-rh) [inline getting_started/access_your_services.adoc]: s/you're/you are

File: getting_started/access_your_services.adoc [hunk 4]
```diff
@@ -113,6 +113,74 @@ Alternatively, you can use a custom domain:
...
+You can alternatively expose your applications internally only, enabling access only through AWS VPC Peering or a VPN connection.
+
+----
+$ oc expose dc httpd-example --type=LoadBalancer --name=internal-lb --dry-run -o yaml | awk '1;/metadata:/{ print "  annotations:\n    service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0" }' | oc create -f -
+service/internal-lb exposed
+----
+
+=== Use your TCP Service
+
+Once you've created your LoadBalancer service, you can access your service by using the URL provided to you by {product-title}.
```
Comment (ahardin-rh) [inline getting_started/access_your_services.adoc]: Once your LoadBalancer service is created, you can

File: getting_started/access_your_services.adoc [hunk 5]
```diff
@@ -113,6 +113,74 @@ Alternatively, you can use a custom domain:
...
+The `LoadBalancer Ingress` value is a URL unique to your service that remains static as long as the service is not deleted. You
+may create a CNAME DNS record for this URL if you prefer to use a custom domain.
```
Comment (ahardin-rh) [inline getting_started/access_your_services.adoc]: If you prefer to use a custom domain, you can create a CNAME DNS record for this URL.

File: getting_started/access_your_services.adoc (hunk 1 above)
Comment (wgordon17) [reply getting_started/access_your_services.adoc]: That seems very odd...is it just because I'm `xref`ing the `adoc` version?

Comment (ahardin-rh) [reply getting_started/access_your_services.adoc]: @wgordon17 Yeah, for some reason the missing section IDs cause the builds to go wonky.

Comment (wgordon17) [reply getting_started/access_your_services.adoc]: Weird 🤷‍♂️

**Comment by wgordon17** [comment]
Thanks for the review @ahardin-rh, all fixed up 👍

File: getting_started/access_your_services.adoc (hunk 1 above)
Comment (ahardin-rh) [reply getting_started/access_your_services.adoc]: I know :-1:

**Comment by ahardin-rh** [comment]
@wgordon17 Do we have a target date for when this should merge?

**Comment by wgordon17** [comment]
@ahardin-rh Ready for this to go live 👍

**Pull request merged**